src/main/java/dan200/computercraft/client/gui/NoTermComputerScreen.java

```diff
@@ -43,6 +48,17 @@ public T getMenu()
     @Override
     protected void init()
     {
+        this.passEvents = true; // to allow gui click events pass through mouseHelper protection (see MouseHelper.OnPres:105 code string)
+        if (mineScreen != null) // if reflection failed - pocket pc still would work but with no mouse move mechanic
+        {
+            minecraft.mouseHandler.grabMouse();
+            try {
+                mineScreen.set(minecraft, this);
```

Comment:
Instead of using reflection, it's probably better to use an access transformer to make the field public. You can add `public net.minecraft.client.Minecraft field_71462_r # currentScreen` to `src/main/resources/META-INF/accesstransformer.cfg`, rebuild and then just set the field normally.

Reply:
Done.